src/main/java/org/testng/reporters/jq/Main.java

```diff
@@ -18,10 +18,11 @@
 
 public class Main implements IReporter {
   private static final String[] RESOURCES = new String[] {
-    "jquery-1.7.1.min.js", "testng-reports.css", "testng-reports.js",
-    "passed.png", "failed.png", "skipped.png", "navigator-bullet.png",
-    "bullet_point.png", "collapseall.gif"
```

Comment:
Yeah I had a look at that and was concerned as well. I originally changed the contents in the `header` file so that it included new relative paths.

I'll make a change to strip out the path names during the copy then, this seems a lower risk if there are assumptions about the location of the files elsewhere